src/Controller/Backend/BackendBase.php

```diff
@@ -71,6 +71,7 @@ public function before(Request $request, Application $app, $roleRoute = null)
         // Sanity checks for doubles in in contenttypes. This has to be done
         // here, because the 'translator' classes need to be initialised.
         $app['config']->checkConfig();
+        $app['config']->checkTaxonomy();
```

Comment:
Not sure yet, was mid-wife-night and just noticed this walking past my screen … let me look post- :coffee: